dotCMS/src/main/java/com/dotmarketing/webdav/DotWebdavHelper.java

```diff
@@ -1313,6 +1313,7 @@ private String deleteSpecialCharacter(String fileName) throws IOException {
 			fileName = fileName.replace("<", "");
 			fileName = fileName.replace(">", "");
 			fileName = fileName.replace("|", "");
+			fileName = fileName.replace("+", " ");
```

Comment:
just to double check, it seems the rest of the replaces, replace with "" and you are replacing with " ".
Just wondering if it is ok

Reply:
ok